### PR TITLE
Add C++ namespaces.

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: thrift/agent.thrift [hunk 1]
```diff
@@ -23,6 +23,7 @@
 include "jaeger.thrift"
 include "zipkincore.thrift"
 
+namespace cpp uber.jaeger.agent.thrift
```
Comment (yurishkuro) [inline thrift/agent.thrift]: maybe `s/uber.jaeger/jaegertracing/`?

Comment (isaachier) [reply thrift/agent.thrift]: Ya probably same applies to https://github.com/isaachier/cpp-client

File: thrift/sampling.thrift [hunk 2]
```diff
@@ -20,6 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+namespace cpp jaegertracing.thrift.sampling_manager
```
Comment (yurishkuro) [inline thrift/sampling.thrift]: inconsistent with agent, it had thrift as a suffix

File: thrift/zipkincore.thrift [hunk 3]
```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+namespace cpp twitter.zipkin.thrift
```
Comment (yurishkuro) [inline thrift/zipkincore.thrift]: is this one necessary? I don't suppose you're planning to use zipkin model in C++ client.

File: thrift/sampling.thrift (hunk 2 above)
Comment (isaachier) [reply thrift/sampling.thrift]: OK sure

File: thrift/zipkincore.thrift (hunk 3 above)
Comment (isaachier) [reply thrift/zipkincore.thrift]: Honestly no. I can revert this. Just did this upfront and never went back to check what was used.

**Comment by isaachier** [comment]
Actually, I need the Zipkin namespace in order to implement the Agent interface method `emitZipkinBatch` (even though all it does is throw an exception).

**Pull request merged**